Client/game_sa/CEntitySA.cpp

```diff
@@ -628,6 +650,33 @@ bool CEntitySA::SetBoneRotation(eBone boneId, float yaw, float pitch, float roll
     return false;
 }
 
+bool CEntitySA::SetBoneRotationQuat(eBone boneId, float x, float y, float z, float w)
+{
+    RpClump* clump = GetRpClump();
+    if (clump)
+    {
+        // updating the bone frame orientation will also update its children
+        // This rotation is only applied when UpdateElementRpHAnim is called
+        CAnimBlendClumpDataSAInterface* clumpDataInterface = *pGame->GetClumpData(clump);
+        AnimBlendFrameData*             frameData = clumpDataInterface->GetFrameDataByNodeId(boneId);
+        if (frameData)
+        {
+            RtQuat* boneOrientation = &frameData->m_pIFrame->orientation;
+            boneOrientation->imag.x = x;
+            boneOrientation->imag.y = y;
+            boneOrientation->imag.z = z;
+            boneOrientation->real = w;
+            CEntitySAInterface* theInterface = GetInterface();
+            if (theInterface)
+            {
+                theInterface->bDontUpdateHierarchy = false;
+            }
```

Comment:
You can remove braces

Reply:
again - im not here refactoring mta repository but adding new functionalities, it's like this everywhere, you can always create own pull request to refactor code

Reply:
When adding new functionality you **must** follow the guidelines. Period.
If you dont like it, dont post pull requests that dont satisfy the requirements

Reply:
> again - im not here refactoring mta repository but adding new functionalities, it's like this everywhere, you can always create own pull request to refactor code

This is your PR, your changes, so it's your responsibility to make adjustments and corrections. Coding guidelines must be followed. No one is going to fix this for you. No one is asking you to refactor MTA's code, just to adhere to the guidelines. So, if you're editing a part of the code, you can also take the opportunity to adjust it according to the guidelines, especially if it's your own code. You can't have this attitude towards others, because if you do, your PRs are likely to be delayed due to refusal to cooperate and implement changes